diff --git a/node_modules/isomorphic-git/index.js b/node_modules/isomorphic-git/index.js
index 8d9845f..47c5887 100644
--- a/node_modules/isomorphic-git/index.js
+++ b/node_modules/isomorphic-git/index.js
@@ -8516,10 +8516,13 @@ async function mergeBlobs({
     format,
     markerSize,
   });
+  /*
+   * @Yiyi: Don't throw error here
   if (!cleanMerge) {
     // all other types of conflicts fail
     throw new MergeNotSupportedError()
   }
+  */
   const oid = await _writeObject({
     fs,
     gitdir,
